controllers/azuremanagedcontrolplane_controller_test.go

```diff
@@ -146,6 +148,9 @@ func TestAzureManagedControlPlaneReconcilePaused(t *testing.T) {
 				SubscriptionID: "something",
 			},
 			ResourceGroupName: name,
+			FleetsMember: &infrav1.FleetsMember{
+				Name: name,
```

Comment:
I think that the unit test is not testing the member name defaulting logic. it's always setting the name to the cluster name.

Reply:
This controller test skips the webhook defaulting and validation so it will not be defaulted here. I will add a separate test for the webhook.